docs/workflow/testing/libraries/testing.md

```diff
@@ -1,54 +1,81 @@
 # Testing Libraries
 
-We use the OSS testing framework [xunit](https://github.com/xunit/xunit).
+## Full Build and Test Run
 
-To build the tests and run them you can call the libraries build script. For libraries tests to work, you must have built the coreclr or mono runtime for them to run on.
+These example commands automate the test run and all pre-requisite build steps in a single command from a clean enlistment.
 
-**Examples**
-- The following shows how to build only the tests but not run them:
+- Build all pre-requisites (clr + libs) and tests and run them using CoreCLR:
 ```
-build.cmd/sh -subset libs.tests
+build.cmd/sh -subset clr+libs+libs.tests -test
```

Comment:
I cloned a brand new repo and ran only this command. It is still in progress, but it appears to running successfully without additional arguments such as -build or -restore.

Reply:
Yes, just realized that -restore -build is implicit when passing in -test: https://github.com/dotnet/runtime/blob/f891033db5b8ebf651176a3dcc3bec74a217f85e/eng/build.ps1#L222-L224. It would not run if you also pass in `-testnobuild`.